### PR TITLE
medical wrench no longer dupes on suicide

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
fix: medical wrench no longer dupes on suicide
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
this isnt a fix but 👍

**Comment by Ghommie** [comment]
Do not desecrate this holy relic like that without providing a good reason at least, please. :-1:

**Comment by Tumesuo** [comment]
I will abuse the shit out of this before its merged

File: code/game/objects/items/tools/wrench.dm [hunk 1]
```diff
@@ -53,22 +59,12 @@
...
 		user.dropItemToGround(W)
-
-	var/obj/item/wrench/medical/W = new /obj/item/wrench/medical(loc)
-	W.add_fingerprint(user)
-	W.desc += " For some reason, it reminds you of [user.name]."
-
-	if(!user)
-		return
-
+	suicider = user.name
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/tools/wrench.dm]:
```suggestion
	suicider = user.real_name
```

**Comment by Fikou** [comment]
cringe change, the medical wrench cult should still exist

**Comment by Ty-the-Smonk** [comment]
Isnt this a feature

File: code/game/objects/items/tools/wrench.dm [hunk 2]
```diff
@@ -53,22 +59,12 @@
...
 	sleep(20)
-
 	if(!user)
 		return
-
 	for(var/obj/item/W in user)
 		user.dropItemToGround(W)
-
-	var/obj/item/wrench/medical/W = new /obj/item/wrench/medical(loc)
-	W.add_fingerprint(user)
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/tools/wrench.dm]: forensics nerf smh

Comment (Tlaltecuhtli) [reply code/game/objects/items/tools/wrench.dm]: arent items logged by default? there is no need for double logging

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/tools/wrench.dm]: ur right

**Comment by zeskorion** [comment]
Do not merge this. Medical wrench suicide is blessed

**Pull request merged**